test/kpod-mount.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+    cleanup_test
+}
+
+IMAGE="alpine:latest"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT ${STORAGE_OPTS}"
+
+@test "mount" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} unmount $ctr_id
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
+    echo "$output"
+    [ "$status" -eq 0 ]
+    root="$output"
+    touch $root/foobar
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} unmount $ctr_id
+    [ "$status" -eq 0 ]
+    stop_crio
+    cleanup_ctrs
+    cleanup_pods
```

Comment:
Both `cleanup_ctrs` and `cleanup_pods` use `crioctl` to communicate with the daemon, so you likely need to call `stop_crio` after calling them.